amazon-kinesis-client-multilang/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
```

Comment:
can u also add small description to pull requests about change, which can help anyone know what this pull request has change for.